test/distributed/org/apache/cassandra/distributed/test/ring/BootstrapTest.java

```diff
@@ -196,6 +197,12 @@ public void bootstrapUnspecifiedFailsOnResumeTest() throws Throwable
                                bootstrap()),
                         newInstance.config().num());
         }
+        catch (AssumptionViolatedException ave)
+        {
+            // We get an AssumptionViolatedException if we're in a test job configured w/vnodes
+            if (ave.getMessage().contains("vnode is requested but not supported"))
```

Comment:
remove this check, the type is very explicit "junit ignore this test", you trying to ignore that isn't good and will be brittle.

Also, add `throw ave` so it actually is skipped and not passing.